mne/viz/ica.py

```diff
@@ -130,6 +130,223 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
+    ax = list()
+    ax.append(fig.add_axes([0.05, 0.5, 0.3, 0.45], label='topo'))
+    ax.append(fig.add_axes([0.5, 0.6, 0.45, 0.35], label='image'))
+    ax.append(fig.add_axes([0.5, 0.5, 0.45, 0.1], label='erp'))
+    ax.append(fig.add_axes([0.05, 0.1, 0.35, 0.3], label='spectrum'))
+    ax.append(fig.add_axes([0.5, 0.1, 0.45, 0.25], label='variance'))
+    return fig, ax
+
+
+def plot_properties(inst, ica=None, picks=None, axes=None, dB=False,
+                    cmap=None, plot_std=True, topo_kws=None, image_kws=None,
+                    show=True):
+    """Display component properties: topography, epochs image, ERP/ERF,
+    power spectrum and epoch variance.
+
+    Parameters
+    ----------
+    inst: instance of Epochs or Raw
+        The data to use in plotting properties.
+    ica : instance of mne.preprocessing.ICA
+        The ICA solution.
+    picks : int | array-like of int | None.
+        The components to be displayed. If None, plot will show the
+        sources in the order as fitted. If more than one components were
+        chosen in the picks - each one will be plotted in a separate figure.
+        Defaults to None.
+    axes: list of matplotlib axes | None
+        List of five matplotlib axes to use in plotting: [topo_axis,
+        image_axis, erp_axis, spectrum_axis, variance_axis]. If None a new
+        figure with relevant axes is created. Defaults to None.
+    dB: bool
+        Whether to plot spectrum in dB. Defaults to False.
+    cmap: matplotlib colormap | None
+        Colormap to use in both topoplot and epochs image. If None topoplot
+        and epochs image use their default "RdBu_r" colormap. Defaults to None.
+    plot_std: bool | float
+        Whether to plot standard deviation in ERP/ERF and spectrum plots.
+        Defaults to True, which plots one standard deviation above/below.
+        If set to float allows to control how many standard deviations are
+        plotted. For example 2.5 will plot 2.5 standard deviation above/below.
+    topo_kws : dict | None
+        Dictionary of arguments to plot_topomap. If None - doesn't pass any
+        additional arguments. Defaults to None.
+    image_kws : dict | None
+        Dictionary of arguments to plot_epochs_image. If None - doesn't pass
+        any additional arguments. Defaults to None.
+    show : bool
+        Show figure if True.
+
+    Returns
+    -------
+    fig : instance of pyplot.Figure
+        The figure.
+
+    """
+    from .epochs import plot_epochs_image
+    from ..io.base import _BaseRaw
+    from ..epochs import _BaseEpochs
+    from ..preprocessing import ICA
+    from ..time_frequency.psd import psd_multitaper
+
+    # current component/channel (with future interactivity in mind)
+    idx = 0
+
+    if not isinstance(inst, (_BaseRaw, _BaseEpochs)):
+        raise ValueError('inst should be an instance of Raw or Epochs,'
+                         ' got %s instead.' % type(inst))
+    elif not isinstance(ica, ICA):
```

Comment:
Actually I think it should be ``if ica is not None and not isinstance(ica, ICA)``